cmd/loki/main.go

```diff
@@ -21,6 +21,10 @@ import (
 	"github.com/grafana/loki/pkg/validation"
 )
 
+func exit(code int) {
+	util_log.Flush()
```

Comment:
I think you're missing the actual exit here?

Reply:
👋🏻